packages/rustcat/PKGBUILD

```diff
@@ -29,6 +28,7 @@ build() {
 package() {
   cd $pkgname
 
-  install -Dm 755 "target/release/$_binname" "$pkgdir/usr/bin/$_binname"
+  install -Dm 755 "target/release/rcat" -t "$pkgdir/usr/bin/"
+  install -D LICENSE -t "$pkgdir/usr/share/licenses/pkgname"
```

Comment:
you have dropped the final `$` before pkgname

Reply:
yeah, corrected